readme.md

```diff
@@ -282,15 +320,15 @@ foods.forEach(function(food){
 })
 ```
 
-If there is more than one argument to the anonymous function, wrap
+If there is more or fewer than one argument to the anonymous function, wrap
```

Comment:
This line might be confusing. Consider `Just like in ES5 we wrap our arguments in parens.`

(original foods.forEach example)

`However, when an ES6 function has only one parameter defined, we're allowed to omit the parens`

(supply single param function example, something like the following)

```js
let myLogger = message => console.log(`Best console log ever!: ${message}`)
```